port_ocean/config/settings.py

```diff
@@ -73,6 +76,26 @@ class IntegrationConfiguration(BaseOceanSettings, extra=Extra.allow):
     integration: IntegrationSettings = IntegrationSettings(type="", identifier="")
     runtime: Runtime = "OnPrem"
 
+    @root_validator()
+    def validate_integration_config(cls, values: dict[str, Any]) -> dict[str, Any]:
+        if not (config_model := values.get("_integration_config_model")):
```

Comment:
I don't fully get the full logic of the method, is the function is only responsible for validation? or it also sets the `integration_config` why is that?